packages/attestation-service/migrations/20200903211921-attestation-v3.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    try {
+      await queryInterface.removeColumn('Attestations', 'smsProvider')
+
+      await queryInterface.addColumn('Attestations', 'countryCode', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      })
+
+      await queryInterface.addColumn('Attestations', 'phoneNumber', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      })
+
+      await queryInterface.addColumn('Attestations', 'message', {
```

Comment:
the message needs persisting because a separate instance may receive the callback saying that delivery failed, and then need to resend the next SMS to the next provider

Reply:
I did consider removing the contents of the message on eventual delivery success or failure (running out of retries).  The number and message are never returned to clients.

Reply:
to clarify -- the separate instances will be expected to share the DB then?

Reply:
yes, exactly

Reply:
I will add a section to the docs explaining this architecture